cmd/lotus-provider/web/api/routes.go

```diff
@@ -2,11 +2,11 @@
 package api
 
 import (
-	"github.com/filecoin-project/lotus/cmd/lotus-provider/deps"
 	"github.com/filecoin-project/lotus/cmd/lotus-provider/web/api/debug"
+	"github.com/filecoin-project/lotus/lib/harmony/harmonydb"
 	"github.com/gorilla/mux"
 )
 
-func Routes(r *mux.Router, deps *deps.Deps) {
-	debug.Routes(r.PathPrefix("/debug").Subrouter(), deps)
+func Routes(r *mux.Router, db *harmonydb.DB) {
```

Comment:
Please don't do this. I'm hoping to support a full set of APIs off of this server eventually.